parser-library/nt-headers.h

```diff
@@ -55,6 +55,49 @@ const std::uint16_t DIR_IAT = 12;
 const std::uint16_t DIR_DELAY_IMPORT = 13;
 const std::uint16_t DIR_COM_DESCRIPTOR = 14;
 
+// Machine Types
```

Comment:
In C++ const global variables have internal linkage by default, which is same as static would do.
I think declaring the variables with constexpr would be a bit more efficient since it enforces compile-time expression evaluation.
If that's ok, I would like to change const to constexpr for all of the variables in separate patch. I think adding static would not have any effect.